ingredients.jsx

```diff
@@ -63,6 +63,14 @@ var Ingredients = React.createClass({
           { this._wholeNumberRoundUpIfAppropriate(this.props.req.lines) }
```

Comment:
Looks like you didn't remove the original adding of line_required here. I suspect you did and this came back due to a merge conflict with my changes to rename _wholeNumberRoundUpIfAppropriate()